src/main/proto/core/usubscription/v3/usubscription.proto

```diff
@@ -69,14 +69,14 @@ service uSubscription {
 
   //  API called by producers to register a topic. This API
   // informs the Subscription Service that to create the topic and it is ready to publish.
-  rpc CreateTopic(CreateTopicRequest) returns (uprotocol.v1.UStatus) {
+  rpc CreateTopic(CreateTopicRequest) returns (PublisherState) {
```

Comment:
There is already INVALID_ARGUMENT and OK vs FAILED so why do we need to send different states? Furthermore this would be breaking changes (again) and would make the APIs not portable with what we deployed in v2.